lib/sass/calcite-web/patterns/_side-nav.scss

```diff
@@ -6,51 +6,50 @@
 
 @mixin side-nav() {
   @include box-sizing(border-box);
-  padding: 0;
-  border: 1px solid $lightest-gray;
-  border-top: none;
-  @include avenir-regular();
-  background-color: $white;
 }
 
-  @mixin side-nav-title() {
-    margin: 0;
-    padding: $baseline/3;
-    @include avenir-regular();
-    @include font-size(-1);
-    background-color: $off-white;
-    border-top: 1px solid $lightest-gray;
+@mixin side-nav-element-base() {
+  @include font-size(-2);
+  @include transition(all 0.15s ease-in-out);
+  display: block;
+  color: $dark-gray;
+}
+
+@mixin side-nav-title() {
+  @include side-nav-element-base();
+  margin: 0 0 -1px 0;
+  padding: $baseline/2 0;
+  border-bottom: 1px solid $lightest-gray;
+  font-weight: 600;
+  word-wrap: break-word;
+  cursor: default;
+
+  + nav .side-nav-link:first-child {
```

Comment:
This is kind of weird. None of the other links have a hover at all, so why does the first one? When I delete these styles it looks fine locally, but maybe there is some other problem you're solving for?

Reply:
Good catch! That's extraneous - it was being used for a "full-width-border" effect on items beneath the title, however we removed that bg-background hover style, so we don't need this anymore.